### ISSUE TITLE
showState = false doesn't work properly

### ISSUE THREAD

**Issue by AKhashi** [altafc22/csc_picker#39]
Hi,

I need to able to show only Countries and Cities lists but I found that showState = true/false flag does not work properly:
 - if to set layout: Layout.vertical then State dropdown list is shown regardless to the value of showState
 - if layout is not set and showState: false then Countries doprdown list is shown only (even if showCities is true);

I took a look on your source code in csc_picker.dart and found that when CSC widget is being built there is no check of showState/showCities at allwhen layout is vertical, and when layout is not set it shows Cities list only if showState is true.

**Comment by latestbob** [comment]
Exactly,, am currently using it now,, I want to exclude the city,, but showCity false even disabled the state dropdown

**Comment by bipindubey-technoark** [comment]
> Exactly,, am currently using it now,, I want to exclude the city,, but showCity false even disabled the state dropdown

Hii @latestbob 
state dropdown gets disabled because you have not selected country 
once country is selected then state dropdown should work.

and we should have one more field as disabledState which will be boolean and then we can use it as per our requirement
@altafc22

**Comment by mrsrmn** [comment]
@altafc22 thank you for being a very responsive developer and not fixing this in 3 years
